app/controllers/application_controller.rb

```diff
@@ -106,8 +106,8 @@ def set_view_paths
     prepend_view_path 'app/views/shinycms'
 
     # Add the default templates directory for any loaded plugins above that
-    Plugin.loaded.each do |plugin_name|
-      prepend_view_path "plugins/#{plugin_name}/app/views/#{plugin_name.underscore}"
+    Plugin.loaded.each do |plugin|
+      prepend_view_path "plugins/#{plugin.name}/app/views/#{plugin.name.underscore}"
```

Comment:
Consider extracting the views path `"plugins/#{plugin.name}/app/views/#{plugin.name.underscore}"` in to a method of `Plugin`